example_FHIR_bundle_message.json

```diff
@@ -0,0 +1,1054 @@
+Sp�ASr�)�x-opt-jms-destQ�x-opt-jms-msg-typeQSs�d�/ID: 4d90b6b6-d609-4158-a88d-89c6b67bd41e: 1: 1: 1-1@�queue: //encounter-report-test@@@@@@�sL*��St�2�messageId�$2B77B3F5-3016-4A6D-821F-152CE420E58DSw��� {
```

Comment:
please remove these characters at the beginning. Let's have json only in this file.